synthesizer/process/src/stack/execute.rs

```diff
@@ -256,9 +266,14 @@ impl<N: Network> StackExecute<N> for Stack<N> {
         let caller = Ternary::ternary(&is_root, request.signer(), &parent);
 
         // Ensure the request has a valid signature, inputs, and transition view key.
-        A::assert(request.verify(&input_types, &tpk, root_tvk, is_root));
+        A::assert(request.verify(&input_types, &tpk, Some(root_tvk.clone()), is_root, call_graph_checksum));
         lap!(timer, "Verify the circuit request");
 
+        // Set the root tvk.
```

Comment:
Is it better to keep these in the original ordering?

Reply:
Up to you.